queryscript/src/error.rs

```diff
@@ -182,10 +182,6 @@ impl<V, E: MultiError> MultiResult<V, E> {
             )),
         }
     }
-
-    pub fn ok<'s>(&'s self) -> &'s V {
```

Comment:
I removed this function because it doesn't actually have `ok()` semantics (i.e. will always return the value, even in the presence of errors)